cmd/limactl/copy.go

```diff
@@ -49,13 +56,20 @@ func copyAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	arg0, err := exec.LookPath("scp")
-	if err != nil {
-		return err
+	defaultTool := Rsync
+	arg0, err := exec.LookPath(string(defaultTool))
+	if err != nil || !strings.HasSuffix(arg0, "rsync") {
+		defaultTool = Scp
```

Comment:
Calling this defaultTool is wrong. This is the actual tool we use. Better to call it copyTool or just tool since we are in the context of the copy command.